version.txt

```diff
@@ -0,0 +1 @@
+18.11.01
```

Comment:
This corresponds to a tag I made on the commit from the last release. If the premise of the PR is abandoned, feel free to delete the tag.